packages/compass-aggregations/src/components/focus-mode/focus-mode-modal-header.tsx

```diff
@@ -47,12 +49,28 @@ const fakeToggleLabelStyles = css({
   )}ch`,
 });
 
+const menuStyles = css({
+  width: '240px',
+});
+
 const menuItemStyles = css({
   '&:after': {
     content: 'attr(data-hotkey)',
+    position: 'absolute',
+    right: 20, // LG has 20px padding on the right
```

Comment:
Or you can set white-space to nowrap and then you don't need remember to change these styles when leafygreen inevitably changes paddings that we don't control 🙂